config/locales/blacklight.en.yml

```diff
@@ -1,3 +1,10 @@
 en:
```

Comment:
I feel like these should go in https://github.com/curationexperts/mahonia/blob/master/config/locales/hyrax.en.yml since there are already similar settings being stored there.